plugins/nexus-repository-pypi/src/test/java/org/sonatype/nexus/repository/pypi/internal/PyPiLinkInfoTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2008-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+
+package org.sonatype.nexus.repository.pypi.internal;
+
+import org.sonatype.goodies.testsupport.TestSupport;
+
+import org.junit.Test;
+
+import java.util.HashSet;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.hasSize;
+
+public class PyPiLinkInfoTest
+        extends TestSupport
+{
+
+    @Test
+    public void should_Ignore_Duplicate_Entries_When_Added_To_A_Set() throws Exception {
+        HashSet<PyPiLinkInfo> set = new HashSet<>();
+
+        set.add(new PyPiLinkInfo("file_a", "link_a"));
+        set.add(new PyPiLinkInfo("file_b", "link_a"));
+        set.add(new PyPiLinkInfo("file_b", "link_b")); // same as previous
```

Comment:
Same as previous ? Previous is B/A, current is B/B ? Do I miss something ?

Reply:
When hashing, I'm considering two `PyPiLinkInfo` the same if they have the same name. This is to drop duplicates which is needed further in the code. However I think now that I think about it since I need to do this at only one location `IndexGroupHandler.mergeResponses` I'll do it manually and keep this class as a simple data container.